manifest.yaml

```diff
@@ -8,6 +8,24 @@ packages:
   - live/grub2-efi-image
   - recovery/cos-img
 
+# Raw disk creation values start
+
+# Where to get the packages from
```

Comment:
maybe we can have as part of a general `raw_disk` section?

Reply:
works for me